utils/emoji_utils.jsx

```diff
@@ -1,16 +1,50 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+const defaultRule = (aName, bName, emojiA, emojiB) => {
+    if (emojiA.category === 'custom' && emojiB.category !== 'custom') {
+        return 1;
+    } else if (emojiB.category === 'custom' && emojiA.category !== 'custom') {
+        return -1;
+    }
+
+    return aName.localeCompare(bName);
+};
+
+const thumbsDownRule = (other) =>
+    (other === 'thumbsup' || other === '+1' ? 1 : 0);
+const thumbsUpRule = (other) => (other === 'thumbsdown' || other === '-1' ? -1 : 0);
```

Comment:
Ternary statements can be a little challenging to follow at first glance, maybe do something like this instead to make the functions a bit clearer? Also, I'm wondering if the function argument should maybe be something like `name`?
``` javascript
const thumbsDownRule = (name) => {
    if (name === 'thumbsup' || name === '+1') {
        return 1;
    }
    return 0;
};
```